custom/icds_reports/utils/aggregation_helpers/complementary_forms.py

```diff
@@ -84,7 +85,8 @@ def aggregation_query(self):
             END AS diet_quantity,
             CASE WHEN ucr.latest_time_end IS NOT NULL
                  THEN ucr.hand_wash ELSE prev_month.hand_wash
-            END AS hand_wash
+            END AS hand_wash,
+            supervisor_id
```

Comment:
generally prefer to put this at the top with state_id (same comment in other places)

Reply:
Sure, why not. Updated.